VEHICLE/AttachVehicleToTrailer.md

```diff
@@ -10,7 +10,8 @@ void ATTACH_VEHICLE_TO_TRAILER(Vehicle vehicle, Vehicle trailer, float radius);
 
 
 ## Parameters
-* **vehicle**: 
-* **trailer**: 
-* **radius**: 
+* **vehicle**: The car to which the trailer is getting attached to
+* **trailer**: The trailer which is getting attached
+* **radius**: Set this always to 1f
```

Comment:
This raises more questions than it answers. Why must it be 1? What effect does it have? Etc.